### PR TITLE
RH-SSO 7.5.0 prep

### PR DESCRIPTION
**Pull request by drichtarik**
@iankko @pskopek

### CODE REVIEW

**Review by iankko — COMMENTED** [review]
@drichtarik First, thank you for the change!

Used the [following helper script](https://github.com/iankko/redhat-sso-7-openshift-image/commit/0c7d691b5b860452b2e055f19a0b5b6bf856ce9a) to produce the version bumps (`7.4` to `7.5`, `74` to `75`, `7.4.7` to `7.5.0`) and comparing the [result it provides](https://github.com/iankko/redhat-sso-7-openshift-image/commit/71265a4985e2bd914fc462241a819955901367b8):

- The only difference seems to be the branch names `sso74-dev` (in my version) vs `sso74-cpaas-dev` (in your version).
- And the additional changes in `image.yaml` in your version (going to inspect those yet):
```
< index d7ceb19..870134d 100644
---
> index 7ea1133..2b99cf6 100644
428,448d427
< @@ -73,17 +73,17 @@ modules:
<            - name: jboss-eap-modules
<              git:
<                    url: https://github.com/jboss-container-images/jboss-eap-modules.git
< -                  ref: EAP_737_CR1
< +                  ref: EAP_750_CR1
<  
<            - name: jboss-eap-image
<              git:
<                    url: https://github.com/jboss-container-images/jboss-eap-7-image.git
< -                  ref: EAP_737_CR1
< +                  ref: EAP_750_CR1
<  
<            - name: cct_module
<              git:
<                    url: https://github.com/jboss-openshift/cct_module.git
< -                  ref: 0.39.3
< +                  ref: 0.41.4
<  
<            - name: sso-modules
<              path: modules
```
The rest of the files seems to be identical between the two versions, meaning  all necessary version strings got updated by your change (AFAICT).

**Comment by drichtarik** [comment]
Is this branch somewhere in the code? BEcause this should be merged into a new sso75-cpaas-dev branch here

File: image.yaml [hunk 1]
```diff
@@ -73,17 +73,17 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_737_CR1
+                  ref: EAP_750_CR1
```
Comment (iankko) [inline image.yaml]: This  should be `EAP_740_CR1` probably (there isn't any tag like `EAP_750_CR1`).

File: image.yaml [hunk 2]
```diff
@@ -73,17 +73,17 @@ modules:
...
 
           - name: jboss-eap-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_737_CR1
+                  ref: EAP_750_CR1
```
Comment (iankko) [inline image.yaml]: And here too. `EAP_740_CR1` from May 11-th seems to be the latest one in https://github.com/jboss-container-images/jboss-eap-7-image/tags.

File: image.yaml [hunk 3]
```diff
@@ -73,17 +73,17 @@ modules:
...
 
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: 0.39.3
```
Comment (iankko) [inline image.yaml:86]: This one is OK.

**Review by iankko — APPROVED** [review]
Latest version LGTM. Thanks, Denis!

**Review by pskopek — APPROVED** [review]
LGTM!

**Pull request merged**